features/FEATURE_BLE/ble/BLETypes.h

```diff
@@ -578,6 +638,13 @@ struct peer_address_type_t :SafeEnum<peer_address_type_t, uint8_t> {
 struct phy_t : SafeEnum<phy_t, uint8_t> {
     /** struct scoped enum wrapped by the class */
     enum type {
+        /**
+         * No phy selected.
+         *
+         * @note This value can be used to indicate the absence of phy
+         */
+        NONE = 0,
```

Comment:
I would keep it as `NONE` as it is a special value. Other members are prefixed with `LE_` because we can't start an identifier with a number (`1M`, `2M`).

What do you think ?

Reply:
That's fine